admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+for your Web server. If you do not install a memcache you will see a warning on 
+your ownCloud admin page. (A memcache is not required and you may ignore the warning if you prefer.)
+
+A PHP opcache stores compiled PHP scripts so they don't need to be re-compiled 
+every time they are called. PHP bundles an opcache since version 5.5, so you 
+don't need to install or configure it.
+
+If you are using PHP 5.4, which is the oldest supported PHP version for 
+ownCloud, you may install the Alternative PHP Cache (APC). This is both an 
+opcache and data cache. APC has not been updated since 2012 and is essentially 
+dead, and PHP 5.4 is very old and lags far behind later releases. If it is possible 
+to upgrade to a later PHP release that is the best option.
+
+Data caching is supplied by the Alternative PHP Cache, user (APCu) in PHP 
+5.5+, Memcached, or Redis.
+
+ownCloud supports multiple memory caching backends, so you can choose the type 
+of memcache that best fits your needs. The supported caching backends are:
+
+* `APC <http://php.net/manual/en/book.apc.php>`_ 
+   A local cache for systems running PHP 5.4.
+* `APCu <https://pecl.php.net/package/APCu>`_ 
+   A local cache for systems running PHP 5.5 and up, APCu 4.06 or better required.
+* `Memcached <http://www.memcached.org/>`_ 
+   Distributed cache for multi-server ownCloud installations.
+* `Redis <http://redis.io/>`_ 
+   For distributed caching, and required for Transactional File Locking.
+
+These are supported but not recommended:
+
+* `XCache <http://xcache.lighttpd.net/>`_ 
+* `ArrayCache <http://www.arbylon.net/projects/knowceans-tools/doc/org/knowceans/util/ArrayCache.html>`_
+   
+Memcaches must be explicitly configured in ownCloud 8.1 and up by installing 
+and configuring your desired cache, and then adding the appropriate entry to 
+``config.php``.
+
+You may use both a local and a distributed cache. Recommended caches are APCu 
+and Redis.
+   
+APC
+---
+
+APC is only for systems running PHP 5.4 and older. The oldest supported PHP 
+version in ownCloud is 5.4.
+
+.. note:: RHEL 6 and CentOS 6 ship with PHP 5.3. See :doc:`php_54_installation`.
+
+This example installs APC on Red Hat/CentOS/Fedora systems running PHP 5.4::
+
+ yum install php-pecl-apc
+ 
+This example install APC on Debian/Ubuntu/Mint systems running PHP 5.4::
+
+ apt-get install php-apc
+
+After installing APC you must restart your Web server. This example restarts 
+Apache on Red Hat/CentOS/Fedora systems::
+
+ systemctl restart httpd.service
+ 
+This example restarts Apache on Debian/Ubuntu/Mint systems::
+
+ service apache2 restart
+ 
+After restarting your Web server, add this line to your ``config.php`` file::
+
+ 'memcache.local' => '\OC\Memcache\APC',
+ 
+Refresh your ownCloud admin page, and the cache warning should disappear.
+
+APCu
+----
+
+PHP 5.5 and up includes an opcache, so you don't need to install or configure 
```

Comment:
Yes, so that statement is wrong. It doesn't include an opcache but it includes an opcode cache called "OPCache".